include/nbl/video/IPhysicalDevice.h

```diff
@@ -605,6 +393,32 @@ class NBL_API2 IPhysicalDevice : public core::Interface, public core::Unmovable
             EQF_SPARSE_BINDING_BIT = 0x08,
             EQF_PROTECTED_BIT = 0x10
         };
+
+        
+        /* QueueFamilyProperties2
+                - VkQueueFamilyCheckpointProperties2NV, VkQueueFamilyCheckpointPropertiesNV: 
+                    These extensions allows applications to insert markers in the command stream and associate them with custom data.
+                    The one with the 2 suffix is provided by VK_KHR_synchronization2 other than VK_NV_device_diagnostic_checkpoints
+                
+                - VkQueueFamilyGlobalPriorityPropertiesKHR 
+                    Related to VK_KHR_global_priority (bool in features)
+                        VK_QUEUE_GLOBAL_PRIORITY_LOW_KHR = 128,
+                        VK_QUEUE_GLOBAL_PRIORITY_MEDIUM_KHR = 256,
+                        VK_QUEUE_GLOBAL_PRIORITY_HIGH_KHR = 512,
+                        VK_QUEUE_GLOBAL_PRIORITY_REALTIME_KHR = 1024,
+                    This extension is basically for querying the queue family's supported global priorities 
+                    In Vulkan, users can specify device-scope queue priorities.
+                    In some cases it may be useful to extend this concept to a system-wide scope.
+                    This device extension allows applications to query the global queue priorities supported by a queue family, and then set a priority when creating queues. The default queue priority is VK_QUEUE_GLOBAL_PRIORITY_MEDIUM_EXT.
+
+
+                - VkQueueFamilyQueryResultStatusProperties2KHR
+                    Related to Queries 
+                    `supported` reports VK_TRUE if query type VK_QUERY_TYPE_RESULT_STATUS_ONLY_KHR and use of VK_QUERY_RESULT_WITH_STATUS_BIT_KHR are supported.
```

Comment:
probably want to expose

Reply:
I'll put a TODO, this is provided by `VK_KHR_video_queue` and needs an extra enum and is kind of an beta extension:
```c
// Provided by VK_VERSION_1_0
typedef enum VkQueryResultFlagBits {
    VK_QUERY_RESULT_64_BIT = 0x00000001,
    VK_QUERY_RESULT_WAIT_BIT = 0x00000002,
    VK_QUERY_RESULT_WITH_AVAILABILITY_BIT = 0x00000004,
    VK_QUERY_RESULT_PARTIAL_BIT = 0x00000008,
#ifdef VK_ENABLE_BETA_EXTENSIONS
  // Provided by VK_KHR_video_queue
    VK_QUERY_RESULT_WITH_STATUS_BIT_KHR = 0x00000010,
#endif
} VkQueryResultFlagBits;
```
https://www.khronos.org/registry/vulkan/specs/1.3-extensions/man/html/VkQueryResultStatusKHR.html